requirements3.txt

```diff
@@ -0,0 +1,17 @@
+# Used by the library (versions most recent, except six mirrors edX current)
+numpy==1.16.4
+scipy==1.2.2
+pyparsing==2.0.7
```

Comment:
pyparsing is at 2.4.0, I think?

Reply:
fixed, pip-installed, and not causing any new breaks in python 3 testing